server/szurubooru/db.py

```diff
@@ -34,3 +34,22 @@ def _bump_query_count() -> None:
 
 
 sa.event.listen(_engine, 'after_execute', lambda *args: _bump_query_count())
+
+import time
+import logging
+ 
+logger = logging.getLogger("myapp.sqltime")
+logger.setLevel(logging.INFO)
+
+def before_cursor_execute(conn, cursor, statement,
+                        parameters, context, executemany):
+    conn.info.setdefault('query_start_time', []).append(time.time())
+    logger.info("Start Query: %s" % statement)
+
+def after_cursor_execute(conn, cursor, statement,
+                        parameters, context, executemany):
+    total = time.time() - conn.info['query_start_time'].pop(-1)
+    logger.info("Total Time: %f" % total)
+
+sa.event.listen(_engine, "before_cursor_execute", before_cursor_execute)
+sa.event.listen(_engine, "after_cursor_execute", after_cursor_execute)
```

Comment:
This should probably be removed or made so that it's only shown when `LOG_SQL` environment variable is set or the `show_sql` parameter in the config.yaml is set